### PR TITLE
Optimize base class

### PR DESCRIPTION
**Pull request by umahaea**
- Optimize Base Class
- Instantiate the pageobjectclass within the test class, Instead of doing that in BaseClass. This reduces lot of lines of code.
- Little bit of clean up in modal tests.
- Remove colorsTest, CalendarTest, NoPlainCSSTest (these are not needed)

### CODE REVIEW

File: src/main/shell_scripts/components.sh [hunk 1]
```diff
@@ -211,30 +211,31 @@ elif [[ $component == "modal" ]]
...
\ No newline at end of file
+echo $TEST_SUITE
+if [[ $TEST_SUITE == "stand_alone" ]]
+then
+install_appHeader master &
+install_contextualHelp master &
+install_drawer master &
+install_avatarDisplay master &
+install_slider master &
+#install_textModal master &
```
Comment (JasonLantz) [inline src/main/shell_scripts/components.sh]: i think i deprecated this when we talked last... you can probably pull it out...

Comment (umahaea) [inline src/main/shell_scripts/components.sh]: yes u did.. I will remove it.

File: src/main/java/utilities/BaseClass.java [hunk 2]
```diff
@@ -122,12 +86,14 @@ protected void setUp() throws MalformedURLException {
                 for (int i = 0; i < (desktopCaps.length - 1); i += 2) {
                     if (platform.startsWith("Windows") && desktopCaps[i].equals("screenResolution")) {
                         caps.setCapability(desktopCaps[i], "2560x1600");
-                        continue;
+                        //continue;
+                    } else if(platform.startsWith("OS X") && desktopCaps[i].equals("screenResolution")){
+                        caps.setCapability(desktopCaps[i], "2048x1536");
                     }
                     caps.setCapability(desktopCaps[i], desktopCaps[i + 1]);
```
Comment (udhadpa) [inline src/main/java/utilities/BaseClass.java]: if my platform fulfills condition 1.
I set  caps.setCapability(desktopCaps[i], "2560x1600");
Then I skip the else if part
and go to caps.setCapability(desktopCaps[i], desktopCaps[i + 1]); again. It's setting it twice.
The same goes with else if part.

File: src/main/java/origamiV2/origamiV2PageObjects/TextModalPageObjects.java [hunk 3]
```diff
@@ -8,15 +8,11 @@
...
-    }
+    public WebDriver driver = null;
+    public AppiumDriver appium = null;
 
-    public TextModalPageObjects(AppiumDriver appium) {
-        this.appium = appium;
+    public TextModalPageObjects() {
     }
 
     public By initiateBtn = By.id("initiatingButton");
```
Comment (udhadpa) [inline src/main/java/origamiV2/origamiV2PageObjects/TextModalPageObjects.java]: This page objects class needs to be removed too

File: src/main/java/compounds/compoundsPageObjects/CompoundsAlertsPageObjects.java [hunk 4]
```diff
@@ -9,16 +9,10 @@
  */
 public class CompoundsAlertsPageObjects {
 
+    public WebDriver driver = null;
```
Comment (udhadpa) [inline src/main/java/compounds/compoundsPageObjects/CompoundsAlertsPageObjects.java]: Do we require these variables in the page object class?

Comment (umahaea) [inline src/main/java/compounds/compoundsPageObjects/CompoundsAlertsPageObjects.java]: This is a place holder Payal. At the moment, we are not using it. If required we will use. For eg. u can refer to pagination page object class.

File: src/main/java/origamiV2/origamiV2PageObjects/TextModalPageObjects.java (hunk 3 above)
Comment (umahaea) [inline src/main/java/origamiV2/origamiV2PageObjects/TextModalPageObjects.java]: nice, I will remove it.

File: src/main/java/utilities/BaseClass.java (hunk 2 above)
Comment (umahaea) [inline src/main/java/utilities/BaseClass.java]: Sorry I didn't notice the "screenResolution", "2048x1536" in the desktopCaps[], I thought u r setting it manually only for windows in the if condition
Very good catch!
I will fix it

**Comment by udhadpa** [comment]
Other than my comments, rest looks good. Awesome work Eajaz 🥇

**Comment by umahaea** [comment]
all approved, so merging it!

**Pull request merged**